lib/ord.js

```diff
@@ -113,14 +112,9 @@ const fGetGroups = (csn, global) => {
     // storing the group ids in a set to avoid duplicates
     let groupIds = new Set();
 
-    let serviceGroups = global.aServices
+    return global.aServices
```

Comment:
I know it's not your problem, but this function implementation is really wired:
1. We should avoid using global to store data. 
2. If we have to use global, we should not pass it as argument, which against the usage of global. 
3. The name 'global.aServices' is really confused. I would suggest that, later we use oop to refactor it. 
4. The function name and return are also confused. The name is fGetGroups, and I would expect to get the list of group. But the return value is indirectly indication. I would suggest, either we define a variable: 'const groupList' and return this variable, or remove line "let groupIds = new Set();", e.g 
```javascript
const fGetGroups = (csn, global) => return global.aServices
 .map((serviceName) => fCreateGroupsTemplateForService(serviceName, csn.definitions[serviceName], new Set()))
 .filter((resource) => resource !== null && resource !== undefined);
```

Reply:
Thx @ductaily, I see https://github.com/cap-js/ord/blob/main/lib/ord.js#L163, the global is the name of object. This is what I want, but the name is conflict with javascript keyword. We should avoid it. And good point, the naming pattern for example **a**Services and **f**GetGroups, we should have agreement in discussion.

Reply:
Just to document something that @zongqichen  and I discussed via call:

I don't think that `global` is a global variable. It's very likely a variable that the CAP Framework passes into the plugins, to pass a global context, where the CAP framework keeps the overall information on what is part of the current application. This is where you find a list of all services, entities etc.